lib/store_web/templates/layout/root.html.leex

```diff
@@ -9,14 +9,21 @@
 
     <%= live_title_tag assigns[:page_title] || "Store", suffix: " ⋅ elementary Store" %>
     <meta name="description" content="Support Development. Get Gear. Win/Win.">
-    <meta name="author" content="elementary, Inc.">
-    <meta name="theme-color" content="#3689e6">
-    <meta name="twitter:card" content="summary_large_image">
+    <meta property="og:description" content="Support Development. Get Gear. Win/Win.">
+
     <meta name="twitter:site" content="@elementary">
     <meta name="twitter:creator" content="@elementary">
-    <meta property="og:description" content="Support Development. Get Gear. Win/Win.">
-    <meta itemprop="description" content="Support Development. Get Gear. Win/Win.">
-    <meta name="apple-mobile-web-app-title" content="elementary">
+
+    <%= if @page_image do %>
```

Comment:
This works for the preview url on product pages, but non product pages won't get the static image. I'm going to rework it in a minute.